test/data_table.go

```diff
@@ -81,10 +80,6 @@ func (table *DataTable) Expand(rootDir string, localRepo *Repo, remoteRepo *Repo
 				templateOnce.Do(func() { templateRE = regexp.MustCompile(`\{\{.*?\}\}`) })
 				match := templateRE.FindString(cell)
 				switch {
-				case match == "{{ root folder }}":
```

Comment:
You can now remove the `rootDir` argument for this function.

Reply:
Nice catch, done